UPGRADE.md

```diff
@@ -2,6 +2,70 @@
 
 ## dev-develop
 
+### Content Types
+
+PropertyParameter are now able to hold metadata. Therefor the Interface has changed. Please check all your ContentTypes
+which uses params.
+
+__Before:__
+
+DefaultParams:
+
+```php
+array(
+    'name' => 'value',
+    ...
+)
+```
+Access in Twig:
+
+```twig
+{{ params.name }}
+```
+
+__After:__
+
+```php
+array(
+    'name' => new PropertyParameter('name', 'string', 'value', array()),
+    ...
+)
+```
+Access in Twig:
+
+```twig
+As String:
+{{ params.name }}
+
+As Array or boolean:
+{{ params.name.value }}
+
+Get translated title:
+{{ params.getTitle('de') }}
```

Comment:
and a better example is `{{ params.testParam }}, {{ params.testParam.name }}, {{ params.testParam.value }}, {{ params.testParam.getTitle('de') }}`